packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -34,7 +34,7 @@ const mapOptionsToQueryParams = (type: EmbedType, embedId: string, options: UrlO
     'embed-hide-footer': options.hideFooter ? 'true' : undefined,
     'embed-hide-headers': options.hideHeaders ? 'true' : undefined,
     'embed-opacity': options.opacity,
-    'disable-tracking': options.disableTracking ? 'true' : undefined,
+    'disable-tracking': options.disableTracking || options.enableSandbox ? 'true' : undefined,
```

Comment:
Please mention this in README in this repo.

Reply:
adding it..